caboose.go

```diff
@@ -206,7 +207,13 @@ func NewCaboose(config *Config) (*Caboose, error) {
 	}
 	if c.config.OrchestratorEndpoint == nil {
 		var err error
-		c.config.OrchestratorEndpoint, err = url.Parse(DefaultOrchestratorEndpoint)
+
+		u := DefaultOrchestratorEndpoint
+		if v := os.Getenv(SaturnOrchUrlEnvKey); len(v) > 0 {
+			u = DefaultOrchestratorCoreEndpoint
+		}
+
+		c.config.OrchestratorEndpoint, err = url.Parse(u)
```

Comment:
why are we adding additional complexity here? we already have an env variable to set `OrchestratorEndpoint` - why wouldn't we pass in the custom 'core only' orchestrator endpoint that way ,versus this pair of over-rides?

in practice, orchestrator is, i think, set in bifrost-gateway which woudl mean neither of these over-rides would take effect?

Reply:
Oh right. I agree. I'll fix this.